src/inputs_from_config.jl

```diff
@@ -88,6 +88,8 @@ function estimands_from_variants(
     return estimands
 end
 
+get_covariates(covariates::Vector, treatments::Tuple) = setdiff(covariates, collect(treatments))
```

Comment:
probably better to make this more explicit, e.g.,  : `get_non_treatment_covariates`. You also don't need to provide types which will restrict the usability of the function.

```julia
get_non_treatment_covariates(covariates, treatments) = setdiff(covariates, collect(treatments))```